pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/phimanifest/PhiManifestService.java

```diff
@@ -158,9 +158,10 @@ public static PhiManifest generateDataForReport(@NonNull ElasticSearchParticipan
         String assentAddendumResponse = convertBooleanActivityAnswerToString(participant.getParticipantAnswerInSurvey(
                 CONSENT_ADDENDUM_PEDIATRICS_ACTIVITY_CODE, SOMATIC_ASSENT_ADDENDUM_QUESTION_STABLE_ID));
         phiManifest.setSomaticAssentAddendumResponse(assentAddendumResponse);
-        Optional<Boolean> consentAnswer = getAdultParticipantConsentedToTumorAnswer(participant, ddpInstanceDto.getStudyGuid());
+        Optional<Object> consentAnswer = getAdultParticipantConsentedToTumorAnswer(participant, ddpInstanceDto.getStudyGuid());
```

Comment:
In this usage, we do want to respond differently to an empty value since it is going to be written to the phi manifest.